canonical-web-and-design/vanilla-framework
# Remove Ubuntu and Canonical social icons

## Done

- Removed Ubuntu and Canonical icons from the social section, as they aren't actually social icons so aren't relevant in this section. If we were to drive users to the social content of Ubuntu or Canonical it would be via Facebook, Twitter, etc...

## QA
- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/patterns/icons/#social and http://0.0.0.0:8101/examples/patterns/icons/icons-social/ 
- You should see that Ubuntu and Canonical are no longer visible

## Details

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2574

## Screenshots

<img width="1439" alt="Screenshot 2019-12-02 at 16 15 59" src="https://user-images.githubusercontent.com/17748020/69976749-67ee5c80-1521-11ea-8512-90ad8a67cbc5.png">
